idunn/places/poi.py

```diff
@@ -13,6 +13,10 @@ class POI(BasePlace):
 
     def __init__(self, d):
         super().__init__(d)
+        if self["id"].startswith("ta"):
```

Comment:
I would even be extra careful in case we have another ID namespace which start with "ta" someday:

```suggestion
        if self["id"].startswith("ta:"):
```